modules/ROOT/pages/deploy-resource-allocation.adoc

```diff
@@ -7,19 +7,49 @@ Before deploying a Mule application to Anypoint Runtime Fabric, determine the co
+
 You can allocate the following resources when deploying an application:
 
 * vCPU cores
+** `Reserved vCPU` - the amount of vCPU guaranteed to the application and reserved for its use.
+** `vCPU Limit` - the maximium amount of vCPU the application can use (the level to which it can burst). This is shared CPU on the worker node. 
 * Memory
 
-Performance information provided in this topic is based on a Runtime Fabric cluster with controller nodes using AWS EC2 M4 instances. M4 instances feature a custom Intel Xeon E5-2676 v3 Haswell processor optimized specifically for EC2. They run at a base clock rate of 2.4 GHz, and can go as high as 3.0 GHz with Intel Turbo Boost. The load generator used was hosted on a separate AWS instance in the same region.
+== vCPU Allocation
+
+When the Reserved vCPU and vCPU limit are equal, the CPU on the worker node is allocated in a guaranteed model. The container always has the specified amount of CPU available. The vCPU is reserved for each replica of the application and is guaranteed regardless of what else is running on the Runtime Fabric.
+
+Guaranteed CPU allocation allows applications to perform consistently by ensuring there is no CPU contention between other applications. It gives guaranteed performance in all cases.
+
+When the `vCPU Limit` is set to a higher value than the `Reserved vCPU`, the application can burst up to the either the vCPU limit value or the amount of unreserved vCPU on the worker node, which ever is lesser.
+
+When using CPU bursting, you must consider the following factors:
+
+* CPU Bursting Limitations
+
+** Minimum reserve: 0.02 vCPU.
+** Minimum limit: 0.07 vCPU. This aligns with the minimum for Runtime Fabric v1.3 and earlier.
+** The CPU limit is bounded by the CPU cores provided on the worker nodes.
+** The maximum suggested utilization per CPU core is 20 - 25 simple apps and API gateways. More complex apps that are frequently used should get more capacity. When the minimum reserve and minimum limit are specified, this provides approximately .5 cores to share across all apps when needed.
+* Runtime Fabric runs a small amount of services on worker nodes. These services consume between 0.3 and 0.5 cores, depending on whether log forwarding is enabled for Anypoint Monitoring.
+* CPU contention. With CPU bursting, applications can compete for a shared resource- the unallocated CPU remaining on the worker node. As more apps are deployed on a worker node, each app has less unallocated CPU to share with other nodes, apps can suffer from poor performance due to not getting enough CPU. The following strategies can be considered for minimizing CPU contention with your applications:
+
+** Deploy intensive nightly batch applications with API that have peak load during the day.
+** Deploy multiple replicas of an application so that they are distributed across worker nodes to maximize the surface area for the unallocated CPU.
```

Comment:
```suggestion
** Deploy multiple replicas of an application so that they are distributed across worker nodes to maximize the surface area for the unallocated CPU.
```